### PR TITLE
Teamsite Official website banner mobile fixes

### PR DESCRIPTION
**Pull request by cehsu**
## Description
These changes address the following:
- [Teamsite: 'Official website banner' text is too big and not wrapping - mobile/tablet media queries](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/14423)
- [Teamsite: 'Official website banner' bug - mobile/tablet media queries](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/14422)

## Testing done
- verified locally

## Screenshots
image

## Acceptance criteria
- [x] Fix Teamsite: 'Official website banner' text is too big and not wrapping - mobile/tablet media queries
- [x] Fix Teamsite: 'Official website banner' bug - mobile/tablet media queries 

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by narin — COMMENTED** [review]
Tested on https://vetsgov-pr-8828.herokuapp.com

Overlapping link (not sure if this is reported in a different bug).  Wrapped text looks good but padding doesn't match provided screenshot.  Is this expected?

screen shot 2018-10-24 at 11 21 39 am

**Comment by cehsu** [comment]
Thanks @narin kindly refer to https://github.com/department-of-veterans-affairs/vets.gov-team/pull/14272 for guidance on testing teamsite-related work locally.

**Review by rianfowler — APPROVED** [review]

File: src/platform/site-wide/sass/consolidated-patches.scss [hunk 1]
```diff
@@ -1,3 +1,20 @@
+.usa-banner-header-expanded {
+  @media (max-width: $medium-screen) {
```
Comment (erikphansen) [inline src/platform/site-wide/sass/consolidated-patches.scss]: Jeff was just talking about how we should try to do our media queries in a mobile-first approach, so using queries that follow the `min-width` pattern instead. I'm not 100% sure that should apply to this teamsite work. But given the nature of these teamsite CSS fixes I'm fine with doing whatever gets the job done.

**Review by erikphansen — APPROVED** [review]

File: src/platform/site-wide/sass/consolidated-patches.scss (hunk 1 above)
Comment (cehsu) [reply src/platform/site-wide/sass/consolidated-patches.scss]: Thanks @erikphansen, this override only applies to smaller screens.

**Pull request merged**